packages/big-design/src/components/Toggle/styled.tsx

```diff
@@ -0,0 +1,44 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+
+import { ToggleLabelProps } from './Toggle';
+
+export const HiddenCheckbox = styled.input`
+  ${hideVisually()}
+`;
+
+export const StyledToggleLabel = styled.label<ToggleLabelProps>`
+  ${withTransition(['background, border-color'])}
```

Comment:
We should should work with design to get the timing and easing function figured out instead of it being a `150ms ease-out`.

Reply:
Yes, in my opinion that can be updated once we get the component checked in. @valterfatia